content/includes/top-nav.html

```diff
@@ -1,4 +1,15 @@
 <header class="header">
+
+  {% if buildtype != 'production' %}
+    <div id="preview-banner" class="usa-alert usa-alert-info">
+      <div class="usa-alert-body">
+        <h3 class="usa-alert-heading">Preview the New VA.gov—Built with Veterans, for Veterans</h3>
+        <p>We're making some improvements to VA.gov. Our new site offers one place to access all VA benefits and health care services. You can sign in with your My HealtheVet, DS Logon, or ID.me account to track your claims, refill your prescriptions, and more.</p>
+        <a href="https://preview.va.gov" class="usa-button-primary">Preview the New VA.gov</a>
```

Comment:
Should this open in a new tab?

Reply:
Good question; I think there's going to be a few updates requested to this banner before the final publish tomorrow, but I'd like to get this out there early for analytics and to hopefully gather a complete set of changes since it's cutting it close.